fiftyone/server/db/interface.py

```
@@ -0,0 +1,188 @@
+            filters: Raw client-supplied sidebar filter dict, with the same
+                semantics as in :meth:`paginate_samples`.
+
+        Returns:
+            A :class:`ValuePickerResult` with the total distinct-value count
+            and the requested page of ``(value, count)`` pairs.
+        """
+        ...
+
+    async def lightning(
+        self,
+        dataset: "fod.Dataset",
```

Ruff (0.15.12)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)